### PR TITLE
Tweak: Added default padding to containers [ED-7593]

### PR DESCRIPTION
**Pull request by hein-obox**

### CODE REVIEW

**Review by udidol — CHANGES_REQUESTED** [review]

File: assets/dev/scss/frontend/_container.scss [hunk 1]
```diff
@@ -70,11 +74,11 @@
 		height: 100%;
 		// Use the Container's `min-height` or default to 100px.
 		min-height: var( --min-height, 100px );
-
+
```
[the remark this answers is not in the document]
Comment (hein-obox) [reply assets/dev/scss/frontend/_container.scss]: fixed

File: assets/dev/scss/frontend/_container.scss [hunk 2]
```diff
@@ -1,5 +1,9 @@
 .e-container {
 	// Set to initial values in order to avoid inheritance which might cause unexpected behavior.
+	--container-default-padding-top: 10px;
+	--container-default-padding-right: 10px;
+	--container-default-padding-bottom: 10px;
+	--container-default-padding-left: 10px;
```
[the remark this answers is not in the document]
Comment (hein-obox) [reply assets/dev/scss/frontend/_container.scss]: @udidol 

Default padding variable: removed. 

`Advanced > Layout > Padding: ` 
- My padding is the default padding css variable: `--padding-top: var( --container-default-padding-top, 10px );`
- The default padding overwritten with the container padding set in the advanced control: `padding-top: var( --padding-top );`

File: core/kits/documents/tabs/settings-layout.php [hunk 3]
```diff
@@ -80,6 +80,25 @@ protected function register_tab_controls() {
 
 		$is_container_active = Plugin::instance()->experiments->is_feature_active( 'container' );
 
+		if ( $is_container_active ) {
+
```
Comment (udidol) [inline core/kits/documents/tabs/settings-layout.php:84]: remove empty line

**Review by udidol — CHANGES_REQUESTED** [review]

File: core/kits/documents/tabs/settings-layout.php (hunk 3 above)
Comment (hein-obox) [reply core/kits/documents/tabs/settings-layout.php]: updated

**Pull request closed without merging**